### PR TITLE
Trial Updates to Nutraceuticals

### PR DESCRIPTION
**Pull request by RLordan**
Hi all, just an update to the nutraceuticals section with new trials added. Feel free to edit and review. @rando2 @yemarshall @nilswellhausen 

On another note, I had written a little bit on probiotics as there are trials now being conducted for COVID. However, probiotics are sometimes a little controversial. If anyone has an opinion on that let me know. I have some literature below for anyone interested.  

https://doi.org/10.1080/07391102.2020.1775123
https://doi.org/10.1016/j.virusres.2020.198018
doi:10.1111/LAM.13334
doi:10.3389/fpubh.2020.00186

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -401,18 +403,22 @@ In influenza, SPM promote antiviral B lymphocytic activities [@doi:10.4049/jimmu
...
+This may be particularly important for patients already receiving pharmaceutical antiplatelet therapies who may become infected.
+This is because there is concern among the medical community that the use of investigational therapeutics on COVID-19 patients already on antiplatelet therapies due to a pre-existing comorbidities may lead to issues with dosing and drug choice, and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031].
 As a result, the use of other therapeutics such as dietary sources of n-3 fatty acids or nutraceuticals with antiplatelet activities may be beneficial and warrant further investigation.
+A recently developed highly purified derivative of EPA known as icosapent ethyl (VascepaTM) [@doi:10.1056/NEJMoa1812792] is currently recruiting participants for randomized controlled clinical trial to investigate its anti-inflammatory activity in COVID-19 positive patients [@clinicaltrials:NCT04412018].
```
Comment (rando2) [inline content/20.treatments.md:413]:
```suggestion
A new clinical trial [@clinicaltrials:NCT04412018] is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified derivative of EPA known as icosapent ethyl (Vascepa ^TM^) [@doi:10.1056/NEJMoa1812792].  
```
Hopefully this is formatted correctly, I haven't done superscripts in Manubot before! I based it on [this](https://github.com/manubot/rootstock/blob/master/content/02.delete-me.md).

File: content/20.treatments.md [hunk 2]
```diff
@@ -401,18 +403,22 @@ In influenza, SPM promote antiviral B lymphocytic activities [@doi:10.4049/jimmu
...
+This is because there is concern among the medical community that the use of investigational therapeutics on COVID-19 patients already on antiplatelet therapies due to a pre-existing comorbidities may lead to issues with dosing and drug choice, and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031].
 As a result, the use of other therapeutics such as dietary sources of n-3 fatty acids or nutraceuticals with antiplatelet activities may be beneficial and warrant further investigation.
+A recently developed highly purified derivative of EPA known as icosapent ethyl (VascepaTM) [@doi:10.1056/NEJMoa1812792] is currently recruiting participants for randomized controlled clinical trial to investigate its anti-inflammatory activity in COVID-19 positive patients [@clinicaltrials:NCT04412018]. 
+Other randomized controlled trials are in the preparatory stages with the intention of investigating the administration of EPA and other bioactive compounds to COVID-19 positive patients to determine if anti-inflammatory effects or disease state improvements are observed [@clinicaltrials:NCT04335032; @clinical trials:NCT04323228].
```
Comment (rando2) [inline content/20.treatments.md:414]:
```suggestion
Other randomized controlled trials are in the preparatory stages with the intention of investigating the administration of EPA and other bioactive compounds to COVID-19 positive patients to determine whether anti-inflammatory effects or disease state improvements are observed [@clinicaltrials:NCT04335032; @clinicaltrials:NCT04323228]. 
```

File: content/20.treatments.md [hunk 3]
```diff
@@ -401,18 +403,22 @@ In influenza, SPM promote antiviral B lymphocytic activities [@doi:10.4049/jimmu
...
+Other randomized controlled trials are in the preparatory stages with the intention of investigating the administration of EPA and other bioactive compounds to COVID-19 positive patients to determine if anti-inflammatory effects or disease state improvements are observed [@clinicaltrials:NCT04335032; @clinical trials:NCT04323228]. 
 Finally, while there have been studies investigating the therapeutic value of n-3 fatty acids against ARDS in humans, there is still limited evidence of their effectiveness [@doi:10.3390/ijms21093104].
 
 **Summary.**
-However, the overall lack of human studies in this area means there is limited evidence as to whether these nutraceuticals could affect COVID-19 infection.  
+The overall lack of human studies in this area means there is limited evidence as to whether these nutraceuticals could affect COVID-19 infection. 
+Consequently, clinical trials have been proposed and are in the preparatory stages to investigate the anti-inflammatory potential of n-3 PUFA and their derivatives for the treatment of COVID-19.
```
Comment (rando2) [inline content/20.treatments.md:419]:
```suggestion
Consequently, clinical trials that have been proposed and are in the preparatory stages will investigate the anti-inflammatory potential of n-3 PUFA and their derivatives for the treatment of COVID-19.   
```

File: content/20.treatments.md [hunk 4]
```diff
@@ -430,11 +436,15 @@ Adequate zinc intake has been associated with reduced incidence of infection [@d
...
 Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
 In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a flourescently tagged SARS-CoV) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
+Currently, there are over ten clinical trials registered with the intention to use zinc in a prevantative or therapeutic manner. 
+Many of these trials have proposed the use of zinc in conjunction with hydroxychloroquine and azithromycin [@clinicaltrials:NCT04370782; @clinicaltrials:NCT04377646; @clinicaltrials:NCT04395768; @clinicaltrials:NCT04373733]. 
+However, it is not known if these trials will change in light of the lack of evidence supporting the use of hydroxychloroquine.
```
Comment (rando2) [inline content/20.treatments.md:441]:
```suggestion
However, it is not known how these trials will respond to the emerging lack of evidence supporting the use of hydroxychloroquine.
```
Just an idea of an alternate way to phrase it, I don't know what you think!

File: content/20.treatments.md [hunk 5]
```diff
@@ -430,11 +436,15 @@ Adequate zinc intake has been associated with reduced incidence of infection [@d
...
+Other trials are investigating zinc in conjunction with other nutrients such as vitamin C or n-3 PUFA [@clinicaltrials:NCT04342728; @clinicaltrials:NCT04323228].
 
 **Summary.**
 
-While overall there is encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus further research is imperative.
-However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelihood of an infection.
+While overall there is encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus the clinical trials that are currently underway will provide vital information on the efficacious use of zinc in COVID-19 prevention and treatment.
```
Comment (rando2) [inline content/20.treatments.md:446]:
```suggestion
Though there is, overall, encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19; thus, the clinical trials that are currently underway will provide vital information on the efficacious use of zinc in COVID-19 prevention and/or treatment. 
```

File: content/20.treatments.md [hunk 6]
```diff
@@ -430,11 +436,15 @@ Adequate zinc intake has been associated with reduced incidence of infection [@d
...
+Other trials are investigating zinc in conjunction with other nutrients such as vitamin C or n-3 PUFA [@clinicaltrials:NCT04342728; @clinicaltrials:NCT04323228].
 
 **Summary.**
 
-While overall there is encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus further research is imperative.
-However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelihood of an infection.
+While overall there is encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus the clinical trials that are currently underway will provide vital information on the efficacious use of zinc in COVID-19 prevention and treatment. 
+However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelihood of an infection
```
Comment (rando2) [inline content/20.treatments.md:447]:
```suggestion
However, given the limited risk, maintaining a healthy diet to ensure an adequate zinc status may be advisable for individuals seeking to reduce their likelihood of infection.
```

File: content/20.treatments.md [hunk 7]
```diff
@@ -460,6 +470,7 @@ As summarized by Carr [@doi:10.1186/s13054-020-02851-4] the trial will not be co
 Another trial in Italy [@clinicaltrials:NCT04323514] intends to deliver a 10 g infusion of vitamin C to 500 severe COVID-19 patients with pneumonia to assess in-hospital mortality over a 72 hr period as the primary outcome.
 The trial is currently recruiting and is due to run until March 2021.
 We will not know how effective vitamin C is as a therapeutic for quite some time due to the length of both trials.
+These are not the only trials investigating the potential value of vitamin C as there are currently over fifteen trials registered with clinicaltrials.gov that are either recruiting or are currently in preperation.
```
Comment (rando2) [inline content/20.treatments.md:473]:
```suggestion
These are not the only trials investigating the potential value of vitamin C, as there are currently (as of June 2020) over fifteen trials registered with clinicaltrials.gov that are either recruiting or are currently in preparation.
```

Comment (rando2) [reply content/20.treatments.md:473]: Just tagging @agitter here, since this seems relevant to #406

File: content/20.treatments.md [hunk 8]
```diff
@@ -499,6 +510,8 @@ Despite these two observational studies potentially linking vitamin D with COVID
 This analysis examined 25-hydroxyvitamin D concentrations in 348,598 UK Biobank participants, of which 449 were SARS-CoV-2 positive. 
 There is significant interest in the link between vitamin D and COVID-19, hence the multitude of studies published in the literature of varying quality and the clinical trials underway.
 One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@clinicaltrials:NCT04334005].
+Other trials are also underway examining various factors including mortality, symptoms recovery, severity of disease, rates of ventilation, inflammatory markers such as C-reactive protein (CRP) and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration [@clinicaltrials:NCT04385940; @clinicaltrials:NCT04334005; @clinicaltrials:NCT04411446; @clinicaltrials:NCT04344041].
```
Comment (rando2) [inline content/20.treatments.md:513]:
```suggestion
Other trials are also underway examining various factors including mortality, symptom recovery, severity of disease, rates of ventilation, inflammatory markers such as C-reactive protein (CRP) and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration [@clinicaltrials:NCT04385940; @clinicaltrials:NCT04334005; @clinicaltrials:NCT04411446; @clinicaltrials:NCT04344041]. 
```

File: content/20.treatments.md [hunk 9]
```diff
@@ -430,11 +436,15 @@ Adequate zinc intake has been associated with reduced incidence of infection [@d
 Similarly, a randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found zinc deficiency to be associated with increased susceptibility to infection and that zinc deficiency could be prevented through supplementation [@doi:10.1093/ajcn/85.3.837].
 Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
 In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a flourescently tagged SARS-CoV) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
+Currently, there are over ten clinical trials registered with the intention to use zinc in a prevantative or therapeutic manner.
```
Comment (rando2) [inline content/20.treatments.md:439]:
```suggestion
Currently, there are over ten clinical trials registered with the intention to use zinc in a preventative or therapeutic manner. 
```

**Comment by rando2** [comment]
@RLordan I might actually be able to help recruit people to critique probiotics text, since I know a lot of gut microbiome researchers! Let me know if you want me to start asking around!

**Comment by RLordan** [comment]
> @RLordan I might actually be able to help recruit people to critique probiotics text, since I know a lot of gut microbiome researchers! Let me know if you want me to start asking around!

Hey @rando2 thanks, I will refine what I've written and add it in. If we think the evidence isn't very compelling then we can just take it out again, it no problem. What I have written mainly just acknowledges that they are being looked into in clinical trials for covid and suggests they have immunomodulatory effects. At this stage it is hard to tell if there is a genuine possibility that they might benefit a patient or if they are being pushed for $$$

**Comment by RLordan** [comment]
@rando2 apologies, clearly my spell check didn't kick in at some points... :/

**Comment by rando2** [comment]
My proposed changes were very minor, so pending no additional comments from anyone, I will commit them and merge at the end of the day today.

**Pull request merged**